calico/felix/config.py

```diff
@@ -441,7 +441,7 @@ def _validate_cfg(self, final=True):
             if parameter.value is None:
                 # No value, not even a default
                 raise ConfigException("Missing undefaulted value",
-                                      self.parameters["InterfacePrefix"])
+                                      name)
```

Comment:
The second parameter to a config exception should be a parameter object, and so `parameter` is the right thing to pass. Always logging "InterfacePrefix" is clearly daft.